service/javadsl/server/src/main/java/com/lightbend/lagom/javadsl/server/ServiceGuiceSupport.java

```diff
@@ -121,9 +270,34 @@ default void bindServices(ServiceBinding<?>... serviceBindings) {
      * @param <T>                   type constraint ensuring <code>serviceImplementation</code> implements <code>serviceInterface</code>
      */
     default <T extends Service> void bindService(Class<T> serviceInterface, Class<? extends T> serviceImplementation) {
-        bindServices(serviceBinding(serviceInterface, serviceImplementation));
+        new LagomServiceBuilder(serviceBinding(serviceInterface, serviceImplementation), this).bind();
     }
 
+    /**
+     * Binds a Service interface with its implementation.
+     * <p>
+     * Inspects the service descriptor and creates routes to serve every call described.
+     * <p>
+     * Allows the configuration of additional Play routers.
+     * <p>
+     * Builds the {@link ServiceInfo} metadata.
+     *
+     * @param serviceInterface      the interface class for a {@link Service}
+     * @param serviceImplementation the implementation class for the <code>serviceInterface</code>
+     * @param additionalRouter      a first AdditionalRouter
+     * @param additionalRouters     other AdditionalRouter if any (can be omitted)
+     * @param <T>                   type constraint ensuring <code>serviceImplementation</code> implements <code>serviceInterface</code>
+     */
+    default <T extends Service> void bindService(Class<T> serviceInterface,
+                                                 Class<? extends T> serviceImplementation,
+                                                 AdditionalRouter additionalRouter,
+                                                 AdditionalRouter... additionalRouters) {
+        new LagomServiceBuilder(serviceBinding(serviceInterface, serviceImplementation), this)
+            .withAdditionalRouters(additionalRouter, additionalRouters)
+            .bind();
+    }
```

Comment:
I'm opting to overload the method because of binary compatibility. We have discussed it before and concluded that it would be OK because that won't impact the user if we add a varargs on the original one. User code will be recompiled and it will work without modification.

However, I don't want to exclude this method in MiMa. I don't think it's a good idea to exclude it because we can accidentally break in future releases without realising.